README.md

```diff
@@ -0,0 +1,171 @@
+
+  <ul>
+    <li>Ruby</li>
+    <li>Rails</li>
+    <li>PostgreSQL</li>
+  </ul>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Store data for the EduPlanner website**
+- **Provide Api endpoints for data Entry**
+- **Provide server side validations**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+This feature has not yet been implemented!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Getting Started
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need:
+  * A basic understanding of Ruby, Rails and Git
+  * A postgresql connection
+  * Create some data before running to see Greatings
+### Usage
+1. Install Ruby
+2. Install Rails
+3. Clone this repo
+```
+git clone https://github.com/ab-noori/EduPlanner-Backend.git
+```
+
+To run the project, execute the following command:
+
+```
+  rails server
+```
+
+To add some greetings to your application run the following commands
+
+``````
+rails db:drop
+rails db:create
+rails db:migrate
+rails db:seed
+``````
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 : **Gisa Chris**
+
+- GitHub: [@gisa chris](https://github.com/gisachris)
+- LinkedIn: [@gisa chris](https://linkedin.com/in/gisa-chris/)
+
+👤 : **Abdulali noori**
+
+- GitHub: [@Abdul Ali Noori](https://github.com/ab-noori)
+- LinkedIn: [@Abdul Ali Noori](https://linkedin.com/in/abdulali-noori)
+
+👤 : **Aimable BANCUNGUYE**
+
+- GitHub: [@Aimable BANCUNGUYE](https://github.com/BANCUNGUYE66)
+- LinkedIn: [@Aimable BANCUNGUYE](https://www.linkedin.com/in/aimable-bancunguye-aba703143/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Google authentication**
+- **More Class based features**
+- **More Student based features**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ab-noori/EduPlanner-Backend/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Please support this project, your contribution can make a significant difference.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Original design idea by [ Murat Korkmaz on Behance.](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## FAQ <a name="faq"></a>
+
+- **Can I use this repository?**
+
+Please feel free to fork this repository for your own projects.
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍